<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Count</class>
 <widget class="QWidget" name="Count">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Count</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelIpv4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   ipv4:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelArp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   arp:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRarp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   rarp:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUdp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   udp:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTcp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   tcp:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelIcmp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   icmp:0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDhcp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>   dhcp:0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
